site/js/dashboard.js

```diff
@@ -0,0 +1,234 @@
+// Copyright OpenSearch Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+const plugins = {
+  'alerting-dashboards-plugin': {
+    name: 'alertingDashboards',
+    videos: [
```

Comment:
We can have an improvement task to scan the test files under the code link https://github.com/opensearch-project/opensearch-dashboards-functional-test/tree/main/cypress/integration/plugins/alerting-dashboards-plugin instead of hardcoding them here.

Reply:
Good idea! https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues/238